presto-docs/src/main/sphinx/installation/deploy-docker.rst

```diff
@@ -0,0 +1,116 @@
+=================================
+Deploy Presto From a Docker Image
+=================================
+
+These steps were developed and tested on Mac OS X, on both Intel and Apple Silicon chips. 
+
+Following these steps, you will:
+
+- install the command line tools for brew, docker, and `colima <https://github.com/abiosoft/colima>`_ tools
+- verify your Docker setup
+- pull the Docker image of the Presto server and the Presto CLI
+- start your local Presto server and Presto CLI
+- query your local Presto server using the Presto CLI
+- query a remote Presto server using the Presto CLI
+
+Installing brew, docker, docker-compose, and colima
+===================================================
+
+This task shows how to install brew, then to use brew to install docker, docker-compose, 
+and colima. 
+
+Note: If you have Docker installed you can skip steps 1-3. It is recommended that you 
+verify your Docker setup by running the command in step 4.
+
+1. If you do not have brew installed, run the following command:
+
+   ``/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"``
+
+2. To install the docker command line and `colima <https://github.com/abiosoft/colima>`_ tools, run the following command:
+
+   ``brew install docker docker-compose colima``
+
+3. Run the following command: 
+
+   ``colima start``
+
```

Comment:
Really appreciate that you bring Colima to my attention since I need an alternative for the docker desktop. I checked its documentation and found maybe it's good to put a note here about the default VM the `start` command creates, in case someone may use this to run some heavy queries :-)
```suggestion
*Note*: The default VM created by Colima has 2 CPUs, 2GiB memory and 60GiB storage. If you need to customize the resources for the VM, check this `doc <https://github.com/abiosoft/colima#customizing-the-vm>`_.
```

But if you think it's too much, you can just ignore this.